### DISCUSSION TITLE
Seeking solutions for basic orders and sales stats

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#15395] [General]
Hi everyone,

Our clients are asking how they can view basic stats related to orders and sales.

Do you know how we can do this?
Are there any built-in features or easy-to-integrate apps?

Thanks and kind regards,

Davide

**Comment by timuric** [comment]
There are many options to setup BI with Saleor, here are few ideas: 
• Export raw order data to a data lake to query it with BI tools
• Export data to Google sheets or Airtable using automation tools like zappier, pipedream etc
• Use mangodb to store json files to create custom reports. Quite lean if you want code first approach.
• For ad hoc sophisticated questions query your database replica. Don't recommend building reports on it as schema is evolving with upgrades. If you decide to pick this route, make sure to make a single SQL view that feeds your reports so you could adjust it in case schema changes
